src/JustSaying/AwsTools/MessageHandling/SnsTopicByName.cs

```diff
@@ -40,7 +41,7 @@ public class SnsTopicByName : SnsTopicBase
             ILoggerFactory loggerFactory,
             SnsWriteConfiguration snsWriteConfiguration,
             IMessageSubjectProvider messageSubjectProvider)
-            : base(serializationRegister, loggerFactory, snsWriteConfiguration, messageSubjectProvider)
+            : base(serializationRegister, loggerFactory, snsWriteConfiguration.HandleException, messageSubjectProvider)
```

Comment:
Nit: `snsWriteConfiguration` needs a null check/guard of some kind as this would NRE.